application/src/test/java/org/opentripplanner/test/support/ResourceLoader.java

```diff
@@ -8,6 +8,7 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URL;
+import java.net.URLDecoder;
```

Comment:
I don't think we need that, right?

Reply:
I'll check again tomorrow, possibly some leftover from a previous try.